docs/pages/enroll-resources/desktop-access/troubleshooting.mdx

```diff
@@ -393,3 +393,34 @@ to force the policy update:
 ```code
 $ gpupdate.exe /force
 ```
+### Shared Directory Error
+
+Attempts to share directory on a remote desktop session results in the error:
+
+```
+Encountered shared directory error 1
+```
+
+**Solution:** Review and disable configured Device Redirection Group Policy
+
+First, make sure that all [Directory sharing](https://goteleport.com/docs/enroll-resources/desktop-access/directory-sharing/#prerequisites) prerequisites has been met.
+
+Teleport requires device redirection policies to be disabled in RDP policies to allow directory sharing from a remote computer in an RDP session
```

Comment:
```suggestion
Teleport requires device redirection to be enabled in order to share a directory from a remote computer in an RDP session.
```

It's confusing, but we need device redirection to be _enabled_, not disabled.
(The group policy setting is a double negative - we are disabling the setting that disables device redirection).